website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -952,6 +952,22 @@ Defines behavior for caching the validation result of previously encountered JWT
 </Tab>
 </Tabs>
 
+## Metrics
+
+The following `envoy` metrics can be used to track JWT authentication details. 
+
+~> **Note:** Envoy does not currently provide any documentation on these metrics.
```

Comment:
This comment is out of date.

Reply:
I thought about this a bit for sure! but we didn't want to explain it ourselves and be completely off based for some of them since there are a few ambiguous ones. we'd rather be transparent about the fact that this is a thing with no docs from envoy that you could use if you want

Reply:
Ya, I think that makes sense. I like the updated wording :ok_hand: